sample/src/main/java/jp/co/cyberagent/android/gpuimage/sample/utils/Camera1Loader.kt

```diff
@@ -0,0 +1,100 @@
+@file:Suppress("DEPRECATION")
+
+package jp.co.cyberagent.android.gpuimage.sample.utils
+
+import android.app.Activity
+import android.hardware.Camera
+import android.util.Log
+import android.view.Surface
+
+class Camera1Loader(val activity: Activity) : CameraLoader() {
+
+    private var cameraInstance: Camera? = null
+    private var cameraFacing: Int = Camera.CameraInfo.CAMERA_FACING_BACK
+
+    override fun onResume() {
+        setUpCamera()
+    }
+
+    override fun onPause() {
+        releaseCamera()
+    }
+
+    override fun switchCamera() {
+        cameraFacing = when (cameraFacing) {
+            Camera.CameraInfo.CAMERA_FACING_FRONT -> Camera.CameraInfo.CAMERA_FACING_BACK
+            Camera.CameraInfo.CAMERA_FACING_BACK -> Camera.CameraInfo.CAMERA_FACING_FRONT
+            else -> return
+        }
+        releaseCamera()
+        setUpCamera()
+    }
+
+    override fun getCameraOrientation(): Int {
+        val degrees = when (activity.windowManager.defaultDisplay.rotation) {
+            Surface.ROTATION_0 -> 0
+            Surface.ROTATION_90 -> 90
+            Surface.ROTATION_180 -> 180
+            Surface.ROTATION_270 -> 270
+            else -> 0
+        }
+        return if (cameraFacing == Camera.CameraInfo.CAMERA_FACING_FRONT) {
+            (90 + degrees) % 360
+        } else { // back-facing
+            (90 - degrees) % 360
+        }
+    }
+
+    override fun hasMultipleCamera(): Boolean {
+        return Camera.getNumberOfCameras() > 1
+    }
+
+    private fun setUpCamera() {
+        val id = getCurrentCameraId()
+        cameraInstance = getCameraInstance(id)
+        if (cameraInstance == null) {
+            Log.e("Camera1Loader", "Camera not found")
+        }
+        val parameters = cameraInstance!!.parameters
```

Comment:
When the `cameraInstance` is null, it should quit this setup.

Reply:
Done